### ISSUE TITLE
Image viewer mode

### ISSUE THREAD

**Issue by lainz** [bgrabitmap/lazpaint#161]
Is possible passing a parameter to lazpaint.exe that instead of editing mode, it displays the image and arrows to navigate from the different images in the folder?

I mean adding another option to the file association like 'View in LazPaint' to act as an image viewer.

**Comment by circular17** [comment]
Sure.

**Comment by lainz** [comment]
Thanks. That's because I use lazpaint as default image viewer.

**Comment by circular17** [comment]
Yes not really convenient if it opens all the windows and menus. Also you cannot navigate the images.

**Comment by lainz** [comment]
Yes that's true. But a button will be good to exit the image viewer and go to edit mode as well. That feature is available in windows image viewer.

**Comment by circular17** [comment]
Ok I understand.

**Comment by c-sanchez** [comment]
A image viewer mode could be cool, my default image viewer is JPEGView (Windows only), this is my favorite because support many image kinds and their interface is minimal but easy to understand with nice functions.
The interface only shows the image, the controls are available but "hidden" like transparent floating buttons which are only shown if you move your mouse cursor over it.

https://sourceforge.net/projects/jpegview/
Here an example of how jpegview works
![jpegview-example01](https://i.ibb.co/R7rf6nT/jpegview-demo01.gif)

**Comment by circular17** [comment]
Wow lot of options. I guess for LazPaint that would rather be: either you look at the image, or you edit the image.

**Comment by Thomassso** [comment]
I would welcome the suggested feature in LazPaint too. This feature exists in XnView, used a lot by myself, but I had to uninstall XnView, because it is free only for private use, not for commercial.

**Comment by lainz** [comment]
With an SSD Drive the default image viewer works fine on Windows, runs fast enough. So for my part the image viewer mode is not needed anymore. 

Still I prefer lazpaint to zoom in and see the pixels I don't like when the image is blurred when I zoom in.

About printing sometimes I use lazpaint because it fits the image perfectly in the paper.

As well I use it as color picker. A good tool for UI design.

So if not used as default image viewer is better anyways in editing mode vs the Windows image viewer and ms paint. Lazpaint wins.

**Comment by sigurd-dev** [comment]
An option for using Lazpaint as an image viewer would be great!

**Comment by circular17** [comment]
I put it as wanted as multiple person showed interest in this mode.
